src/CronJobListBuilder.php

```diff
@@ -99,11 +99,17 @@ public function buildRow(EntityInterface $entity) {
   public function getDefaultOperations(EntityInterface $entity) {
     $operations = parent::getDefaultOperations($entity);
     if ($entity->status() && $entity->isValid()) {
-      $operations += ['run' => [
+      $operations += [
+        'run' => [
         'title' => t('Run'),
         'weight' => 9,
         'url' => $entity->toUrl('run'),
-      ]];
+      ],
+        'logs' => [
```

Comment:
log should also be available if the entity is not enabled. It might still have log entries.